scripts/runtime-code-shasum.sh

```diff
@@ -19,4 +19,5 @@ ${TAR} -c --sort=name --owner=root:0 --group=root:0 --mode 644 --mtime='UTC 2020
     runtime \
     runtime-modules \
     utils/chain-spec-builder \
-    joystream-node.Dockerfile | shasum | cut -d " " -f 1
+    joystream-node.Dockerfile \
+    joystream-node-armv7.Dockerfile | shasum | cut -d " " -f 1
```

Comment:
```suggestion
    node \
    joystream-node-armv7.Dockerfile | shasum | cut -d " " -f 1
```

In addition to the new Dockerfile for armv7, I realized we are also missing the `node/` folder